### ISSUE TITLE
speech to text --> text to speech loop

### ISSUE THREAD

**Issue by aferriss** [dariusk/NaNoGenMo-2015#53]
a novel of content generated by looping a spoken phrase through a speech to text, text to speech feedback loop. 
[See here](http://adamferriss.com/speech/)
[archive](http://adamferriss.com/speech/getPoems.php)

**Comment by hugovk** [comment]
Interesting, a bit like Alvin Lucier's I Am Sitting in a Room.

**Comment by porglezomp** [comment]
For me, it doesn't wait for the input to stop, so it starts spewing out the first word and getting confused right away

**Comment by aferriss** [comment]
@porglezomp yes, it's set to do recognition continuously and produce interim results instead of waiting for phrases to finish. See here
https://developers.google.com/web/updates/2013/01/Voice-Driven-Web-Apps-Introduction-to-the-Web-Speech-API?hl=en
